src/sql/changes/update_2021-01-15_sql_clean.sql

```diff
@@ -0,0 +1 @@
+DROP TYPE project_return;
```

Comment:
where are all the other changes?

Reply:
Functions are automatically dropped and recreated now.  Did I miss any changes that are not just changes to functions?

Reply:
I added `ROP VIEW IF EXISTS project_boundary CASCADE;`